podpac/core/interpolation/nearest_neighbor_interpolator.py

```diff
@@ -283,7 +283,7 @@ def _get_uniform_index(self, dim, source, request, bounds=None):
                 step = self._time_to_float(source.step, source, request)
             else:
                 step = source.step
-            rindex[np.abs(index - rindex) * step > tol] = -1
+            rindex[np.abs(index - rindex) * np.abs(step) > tol] = -1
```

Comment:
I found one more stray absolute value... that's why I wanted to do a thorough review.